Some muxers come with a ',' in their name, replace it by '_' as in the other
places.

Index: gst-libav-1.1.0_pre20130128/ext/libav/gstavmux.c
===================================================================
--- gst-libav-1.1.0_pre20130128.orig/ext/libav/gstavmux.c
+++ gst-libav-1.1.0_pre20130128/ext/libav/gstavmux.c
@@ -937,7 +937,7 @@ gst_ffmpegmux_register (GstPlugin * plug
     p = type_name;
 
     while (*p) {
-      if (*p == '.')
+      if (*p == '.' || *p == ',')
         *p = '_';
       p++;
     }
